src/actions/vstudio/_vstudio.lua

```diff
@@ -352,6 +352,18 @@
 		return links
 	end
 
+---
+-- Assemble the list of library directories.
+--
+-- @param cfg
+--    The active configuration.
+-- @return
+--    The list of library directories, ready to be used in Visual Studio's
+--    AdditionalLibraryDirectories element.
+---
+	function vstudio.getLibraryDirectories(cfg)
+		return table.filterempty(config.getlinks(cfg, "system", "directory"))
+	end
```

Comment:
If you call `vstudio.path()` here, you could get rid of the `table.filterempty()`, and you wouldn't have to call `path.translate()` on the results.